getting-started/setup-building.rst

```diff
@@ -347,148 +349,145 @@ This section explains how to install additional extensions (e.g. ``zlib``)
 on :ref:`Linux <deps-on-linux>` and :ref:`macOS`.  On Windows,
 extensions are already included and built automatically.
```

Comment:
Good idea including the minimal Windows tab, it means the irrelevant stuff is tidied away. Let's avoid this bit of repetition:

![image](https://github.com/python/devguide/assets/1324225/20d760a8-f8eb-4861-9863-f5d7f466214f)


```suggestion
on :ref:`Linux <deps-on-linux>` and :ref:`macOS`.
```